databook/background/Mouse-visual-system.md

```diff
@@ -36,24 +30,22 @@ rodents broadly) lack the orientation columns found in primates and cats. While
 the latter species have neurons that are tuned with similar orientations
 organized in columns, the rodent visual cortex has a "salt and pepper"
 organization, where the orientation selectively of neighboring neurons are
-randomly organized. {reference Ohki 2005}
+randomly organized. {cite:p}`ohki`
 
 Beyond {term}`V1` there are additional higher visual areas (or {term}`HVA`s).
 These are cortical areas that exhibit <b>retinotopy</b> {Marina: can you unpack
 HVAs a bit here?}
 
-An early question was whether neurons in different {term}'HVA's had distinct
+An early question was whether neurons in different HVAs had distinct
```

Comment:
Why remove the glossary reference?

Reply:
because we reference it a line above - just making things a little cleaner.